src/sdk/sql_cluster_router.h

```diff
@@ -190,6 +190,17 @@ class SQLClusterRouter : public SQLRouter {
     std::shared_ptr<hybridse::sdk::Schema> GetTableSchema(
         const std::string& db, const std::string& table_name) override;
 
+    base::Status HandleSQLCreateProcedure(hybridse::node::CreateProcedurePlanNode* plan,
+            const std::string& db, const std::string& sql,
+            std::shared_ptr<::openmldb::client::NsClient> ns_ptr);
+
+    base::Status HandleSQLCreateTable(hybridse::node::CreatePlanNode* create_node, const std::string& db,
```

Comment:
By convention, it should write as `HandleSqlCreateTable`

Reply:
maybe resolve later

Reply:
yep. `SQLXXX` name-style is exists in many places. we will update these in another pr